client/app/models/package.js

```diff
@@ -2,16 +2,36 @@ import Model, { attr, belongsTo } from '@ember-data/model';
 import { inject as service } from '@ember/service';
 import FileManager from '../services/file-manager';
 
+// TODO: Get theses codes from a Constants service
+const PACKAGE_STATUS_CODES = {
+  PACKAGE_PREPARATION: 'Package Preparation',
+  CERTIFIED: 'Certified',
+  FINAL_APPROVAL: 'Final Approval',
+  REVIEWED_NO_REVISIONS_REQUIRED: 'Reviewed - No Revisions Required',
+  REVIEWED_REVISIONS_REQUIRED: 'Reviewed - Revisions Required',
+  WITHDRAWN: 'Withdrawn',
+  SUBMITTED: 'Submitted',
+  UNDER_REVIEW: 'Under Review',
+  MISTAKE: 'Mistake',
+};
+
+const PACKAGE_VISIBILITY_CODES = {
+  APPLICANT_ONLY: 717170002,
+  GENERAL_PUBLIC: 717170003,
+};
+
 export default class PackageModel extends Model {
   ready() {
-    const fileQueue = this.fileQueue.create(this.id);
-
-    this.fileManager = new FileManager(
-      this.id,
-      this.documents,
-      [],
-      fileQueue,
-    );
+    if (this.isApplicantPackage) {
+      const fileQueue = this.fileQueue.create(this.id);
+
+      this.fileManager = new FileManager(
+        this.id,
+        this.documents,
+        [],
+        fileQueue,
+      );
+    }
   }
```

Comment:
Did you see the deprecation warning link I sent? If we're going to refactor this, we should just refactor it away from the deprecated API.

Reply:
Yeah, I wanted to do this in small chunks tho! I agree. 
I looked into using the afterModel hook, but felt it was too many changes in one commit

Reply:
I'm not sure why this change is necessary. We don't need to optimize for this. I remember we talked about this, it's a lot of code to add just to add an optimization that we're not sure is even an issue to begin with.

Reply:
Hmm ok, I can close this.